lib/init/rc.boot

```diff
@@ -167,6 +171,14 @@ main() {
     command -v udevd >/dev/null &&
         udevadm control --exit
 
+    log "Running rc.d scripts..."; {
+        set +f
+        for file in /etc/rc.d/*.boot ; do
```

Comment:
Remove space before `;`.

Reply:
This block *might* be better before `command -v udevd ...` as some scripts may depend on `udev` to be running. I'm not sure if this will ever be the case though.

Reply:
agreed